Content/LeagueSandbox-Scripts/Champions/Yasuo/Q1.cs

```diff
@@ -0,0 +1,79 @@
+using System.Numerics;
+using System.Linq;
+using GameServerCore.Enums;
+using GameServerCore.Domain.GameObjects;
+using static LeagueSandbox.GameServer.API.ApiFunctionManager;
+using GameServerCore.Domain;
+using LeagueSandbox.GameServer.Scripting.CSharp;
+using GameServerCore;
+
+namespace Spells
+{
+    public class YasuoQW : IGameScript
+    {
+        public void OnActivate(IChampion owner)
+        {
+        }
+
+        public void OnDeactivate(IChampion owner)
+        {
+        }
+
+        public void OnStartCasting(IChampion owner, ISpell spell, IAttackableUnit target)
+        {
+            var current = new Vector2(owner.X, owner.Y);
+            var to = Vector2.Normalize(new Vector2(spell.X, spell.Y) - current);
+            var range = to * 475;
+            var trueCoords = current + range;
+
+            FaceDirection(owner, trueCoords, true, 0f);
+        }
+
+        public void OnFinishCasting(IChampion owner, ISpell spell, IAttackableUnit target)
+        {
+            var _hasbuff = owner.HasBuffGameScriptActive("YasuoE", "YasuoE");
+            if (_hasbuff)
+            {
+                spell.SpellAnimation("SPELL3b", owner);
+                AddParticleTarget(owner, "Yasuo_Base_EQ_cas.troy", owner);
+                AddParticleTarget(owner, "Yasuo_Base_EQ_SwordGlow.troy", owner,bone: "C_BUFFBONE_GLB_Weapon_1");

```

Comment:
` owner,bone: "C_BUFFBONE_GLB_Weapon_1"`

If you want to use named parameters, please be consistent.